### DISCUSSION TITLE
tutorials

### DISCUSSION THREAD

**Discussion by sujee** [data-prep-kit/data-prep-kit#760] [Ideas]
I am proposing we have a tutorials section.

Tutorials can be about doing a certain task 

- processing pdfs, involving multiple transforms
- how to score documents
- how to remove PII 
- etc

Tutorials can be part of our github repo and website.

Tutorials typically include

- a working notebook
- an accompanying 'tutorial' that explains concepts / sections of notebook
- may be a video

Here are some  examples 

- https://sparkbyexamples.com/
- and [one i wrote on mongodb](https://www.mongodb.com/developer/products/atlas/quickstart-vectorsearch-mongodb-python/)

**Comment by agoyal26** [comment]
@sujee we are working on creating a course structure for DPK along these lines. Lets connect and identify intersections to avoid duplicate work.
